src/api/app.js

```diff
@@ -18,8 +18,7 @@ module.exports.manifest = {
   getOwnContainer: 'promise',
   getContainer: 'promise',
   reconnect: 'promise',
-  logPath: 'promise',
-  free: 'sync'
```

Comment:
We need to keep all `free` functions except for MutableData.free which doesn't exist anymore.